File: pyhton64/nivel1/exer1.py
quantidade_minima = input("MAX: ")
quantidade_minima = int(quantidade_minima)

quantidade_maxima = input("MIN: ")
quantidade_maxima = int(quantidade_maxima)

estoque_medio = float((quantidade_minima + quantidade_maxima) / 2)
print(f'Estoque médio é de {estoque_medio}')

File: java64/nivel1/exer1.java
import javax.swing.JOptionPane;

public class exer1 {
    public static void main(String[] args) {
        int quantidadeMaxima = 0;
        int quantidadeMinima = 0;
        double media;

        quantidadeMaxima = Integer.parseInt(JOptionPane.showInputDialog(null, "MAX: "));
        quantidadeMinima = Integer.parseInt(JOptionPane.showInputDialog(null, "MIN: "));

        media = (double)(quantidadeMaxima + quantidadeMinima) / 2;
        JOptionPane.showMessageDialog(null, media);
    }
}